website/docs/r/cloud_backup_schedule.html.markdown

```diff
@@ -143,7 +143,10 @@ resource "mongodbatlas_cloud_backup_schedule" "test" {
 * `reference_hour_of_day` - (Optional) UTC Hour of day between 0 and 23, inclusive, representing which hour of the day that Atlas takes snapshots for backup policy items.
 * `reference_minute_of_hour` - (Optional) UTC Minutes after `reference_hour_of_day` that Atlas takes snapshots for backup policy items. Must be between 0 and 59, inclusive.
 * `restore_window_days` - (Optional) Number of days back in time you can restore to with point-in-time accuracy. Must be a positive, non-zero integer.
-* `update_snapshots` - (Optional) Specify true to apply the retention changes in the updated backup policy to snapshots that Atlas took previously.
+* `update_snapshots` - (Optional) Specify true to apply the retention changes in the updated backup policy to snapshots that Atlas took previously. 
+  
+  **Note** This parameter is not updated on return from API
```

Comment:
@martinstibbe this Note is not clear (at least to me). do you mind making more descriptive to help future users?